Lib/ast.py

```diff
@@ -1346,10 +1346,20 @@ def visit_Call(self, node):
                 self.traverse(e)
 
     def visit_Subscript(self, node):
+        def is_simple_tuple(slice_value):
+            # return True if the slice is a non-empty tuple
+            # and there aren't any starred expression inside
+            # of the elements
```

Comment:
Please, explain the *why*. This comment is describing the code verbatim and that does not add much